android/src/main/java/com/reactnativebledidcomm/BleDidcommEvent.kt

```diff
@@ -3,6 +3,9 @@ package com.reactnativebledidcomm
 enum class BleDidcommEvent(val token: String) {
     OnDiscoverPeripheral("onDiscoverPeripheral"),
     OnConnectedPeripheral("onConnectedPeripheral"),
+    OnConnectedCentral("OnConnectedCentral"),
     OnReceivedWriteWithoutResponse("onReceivedWriteWithoutResponse"),
-    OnReceivedNotification("onReceivedNotification")
+    OnReceivedNotification("onReceivedNotification"),
```

Comment:
Maybe these like `"onReceivedNotification"` could come from an enum or sth like that? Not sure if that adds much value though.

Reply:
A bit late, but this is an equivalent enum in kotlin.